csharp.test/PhysicalValueGetter.cs

```diff
@@ -21,17 +20,15 @@ public PhysicalValueGetter(long numValues)
 
             while (columnReader.HasNext)
             {
-                var levelsRead = columnReader.ReadBatch(
-                    _numValues - totalLevels, defLevels.AsSpan(totalLevels), repLevels.AsSpan(totalLevels), values.AsSpan(totalValues), 
-                    out var valuesRead);
+                var levelsRead = columnReader.ReadBatch(_numValues - totalLevels, defLevels.AsSpan(totalLevels), repLevels.AsSpan(totalLevels), values.AsSpan(totalValues), out var valuesRead);
```

Comment:
Same remark as above. The previous style was more readable (`in` and `out` arguments on different lines).